engine-tests/src/main/java/org/terasology/engine/integrationenvironment/Engines.java

```diff
@@ -195,16 +194,31 @@ TerasologyEngine createHeadlessEngine() throws IOException {
     TerasologyEngine createHeadedEngine() throws IOException {
         EngineSubsystem audio = new LwjglAudio();
         TerasologyEngineBuilder terasologyEngineBuilder = new TerasologyEngineBuilder()
-                .add(new WithUnittestModule())
+                .add(new IntegrationEnvironmentSubsystem())
                 .add(audio)
```

Comment:
:question: for the sake of consistency, can we just inline the creation of `audio` here as well, or do we need it explicitly somewhere else?

```suggestion
                .add(new LwjglAudio())
```